src/Services/EngineService.cs

```diff
@@ -59,8 +59,6 @@ private static bool IsBlink(UserAgent agent)
                && agent.Contains(Engine.WebKit);
 
         private static bool IsEdge(UserAgent agent, Platform os)
-            => agent.Contains(Engine.Edge)
-               || agent.Contains("Edg")
-               && (Platform.Windows | Platform.Android).HasFlag(os);
+            => agent.Contains(Engine.Edge);
```

Comment:
There is the case in some Microsoft edge that has only `edg` in the user agent. 

What is your user agent that resolve as difference engine?